services-js/commissions-app/src/server/email/policyOffice.mjml.hbs

```diff
@@ -0,0 +1,44 @@
+<mjml>
+    <mj-head>
+        <mj-font name="Lora" href="https://fonts.googleapis.com/css?family=Lora:400,700,400i" />
+        <mj-font name="Montserrat" href="https://fonts.googleapis.com/css?family=Montserrat:400,700" />
+
+        <mj-attributes>
+            <mj-all font-family="Lora, serif" font-size="16px" line-height="1.8" />
+        </mj-attributes>
+    </mj-head>
+
+    <mj-body>
+        <mj-section padding="25px 0 10px">
+            <mj-column>
+                <mj-image width="200px" padding="0" align="left" alt="City of Boston" src="https://patterns.boston.gov/images/public/logo.png"></mj-image>
+
+            </mj-column>
+        </mj-section>
+
+        <mj-section padding="20px 0 10px">
+            <mj-group>
+                <mj-column width="12%" vertical-align="middle">
+                    <mj-image align="left" padding="0" alt="" src="https://patterns.boston.gov/images/public/seal.png"></mj-image>
+                </mj-column>
+
+                <mj-column width="88%" vertical-align="middle">
+                    <mj-text padding="0 0 0 10px" line-height="1.5">
+                        <p>Hello,</p>
+
+                        <p>A new application was received for the following Boards and Commissions:</p>
```

Comment:
It might be worth putting the applicant name in.

Reply:
@paylakatel what do you think?

Reply:
That would be great if we can! Could we make the language "A new application was received from <first> <last> for the following Boards and Commissions:"

Reply:
Sure can! Would you also like to add “Hello _firstName_,” to the applicant email templates?

Reply:
Yes! Thank you